README.md

```diff
@@ -4,7 +4,7 @@
 ![license](https://img.shields.io/github/license/rastapasta/pokemon-go-mitm-node.svg)
 
 ***IMPORTANT:***
-Niantic introduced certificate pinning in the most recent version of the app. Either downgrade to [0.29.3](https://www.apkmirror.com/apk/niantic-inc/pokemon-go/pokemon-go-0-29-3-release/) or root your phone and use [pokemon-go-xposed](https://github.com/rastapasta/pokemon-go-xposed) to make it work again like smooth whipped cream!
+Niantic introduced certificate pinning in the most recent version of the app. Either ~~downgrade to [0.29.3](https://www.apkmirror.com/apk/niantic-inc/pokemon-go/pokemon-go-0-29-3-release/)~~ (Niantic is blocking 0.29.3 now) or root your phone and use [pokemon-go-xposed](https://github.com/rastapasta/pokemon-go-xposed) to make it work again like smooth whipped cream!
```

Comment:
Probably? Not sure, didn't dig in much myself.

On Thu, Aug 4, 2016 at 1:32 PM, Vitaly Aminev <notifications@github.com>
wrote:

> In README.md
> <https://github.com/rastapasta/pokemon-go-mitm-node/pull/111#discussion_r73597227>
> :
>
> > @@ -4,7 +4,7 @@
> >  ![license](https://img.shields.io/github/license/rastapasta/pokemon-go-mitm-node.svg)
> >
> >  ***IMPORTANT:***
> > -Niantic introduced certificate pinning in the most recent version of the app. Either downgrade to [0.29.3](https://www.apkmirror.com/apk/niantic-inc/pokemon-go/pokemon-go-0-29-3-release/) or root your phone and use [pokemon-go-xposed](https://github.com/rastapasta/pokemon-go-xposed) to make it work again like smooth whipped cream!
> > +Niantic introduced certificate pinning in the most recent version of the app. Either ~~downgrade to [0.29.3](https://www.apkmirror.com/apk/niantic-inc/pokemon-go/pokemon-go-0-29-3-release/)~~ (Niantic is blocking 0.29.3 now) or root your phone and use [pokemon-go-xposed](https://github.com/rastapasta/pokemon-go-xposed) to make it work again like smooth whipped cream!
>
> afaik version is reported via the client. Can it be simply changed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rastapasta/pokemon-go-mitm-node/pull/111/files/9b5148b9fc9f88e3a5bff8e7bcae5baf9fc2c7b0#r73597227>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARlXphf91X0n5xnsjDbFcHWaKA-nHaOks5qckx6gaJpZM4Jc06r>
> .
>